module/modmanager/manager.go

```diff
@@ -836,6 +840,9 @@ func (m *module) startProcess(
 				return rutils.NewModuleStartUpTimeoutError(m.cfg.Name)
 			}
 			return ctxTimeout.Err()
+		case <-slowTicker.C:
+			elapsed := time.Since(startTime).Seconds()
+			logger.Warnf("%q slow startup detected. Elapsed %.2f seconds", m.cfg.Name, elapsed)
```

Comment:
Do we want a similar check in `checkReady`? I'm not sure Naomi's case wouldn't have triggered this log, as the process started it just didn't respond to a ready request.